exporters/cinder.go

```diff
@@ -308,7 +312,52 @@ func ListVolumeLimits(exporter *BaseOpenStackExporter, ch chan<- prometheus.Metr
 		if err != nil {
 			return err
 		}
-
+		jsonLimits := quotasets.GetUsage(exporter.Client, p.ID).PrettyPrintJSON()
```

Comment:
Can you move this entire logic into its own unit/method so you can add a test for this? I think it is too much logic for a single function.

Reply:
Thank you for reviewing!
Moved logic to function in 085a1ae20af7403bcf732e4dc0d2089c129747f3